module/entityAssessors/helper.js

```diff
@@ -952,4 +944,61 @@ module.exports = class EntityAssessorHelper {
         })
     }
 
+
+
+    /**
+      * Bulk create assessments By entityId and role.
+      * @method
+      * @name bulkCreateByUserRoleAndEntity - Bulk create assessments by entity and role.
+      * @param {Object} userAssessmentData - user assessment data
+      * @param {String} userToken - logged in user token.
+      * @returns {Object}  Bulk create user assessments.
+     */
+
+    static bulkCreateByUserRoleAndEntity(userAssessmentData, userId,userToken) {
```

Comment:
@deepakr123 - No need of token here, internal access token only should be required to access the API

Reply:
resolved